docs/understanding-tracking-design/managing-tracking-scenarios-api/index.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "🆕 Managing tracking scenarios via the API"
+date: "2023-05-09"
+sidebar_position: 93
+---
+
+With the **Tracking Scenarios API**, you can efficiently manage tracking scenarios programmatically. Whether you want to retrieve, create, edit, publish, deprecate, or delete tracking scenarios, the API provides the necessary endpoints and functionalities.
+
+Start integrating the Tracking Scenarios API into your workflows and take control of your tracking scenarios in a seamless and automated manner. If you have any questions or need assistance, feel free to reach out to our support team. Happy tracking scenario management!
+
+## Getting Started
+For detailed information about each request, including query parameters and response formats, refer to the corresponding [API documentation](https://console.snowplowanalytics.com/api/msc/v1/docs).
+
+### Authorizing in the API Documentation
+
+To be able to post sample requests in the documentation you need to click the `Authorize` button at the top of the document and authorize with your token. The value for the token field in each individual requests is overwritten by this authorization.
+
+Each request will need to include your company's `organizationID` which is a UUID that can be retrieved from the URL immediately following the .com when visiting console:
```

Comment:
"immediately following the .com" --> "immediately following `.com`"

Reply:
Done